pkg/cloud/aggregation.go

```diff
@@ -8,39 +8,64 @@ import (
 	corev1 "k8s.io/api/core/v1"
 )
 
-var aggregationVarNames = []string{
-	"K6_CLOUD_AGGREGATION_MIN_SAMPLES",
-	"K6_CLOUD_AGGREGATION_PERIOD",
-	"K6_CLOUD_AGGREGATION_WAIT_PERIOD",
-	"K6_CLOUD_METRIC_PUSH_INTERVAL",
-	"K6_CLOUD_MAX_METRIC_SAMPLES_PER_PACKAGE",
-	"K6_CLOUD_MAX_METRIC_PUSH_CONCURRENCY",
+var aggregationVarNames = map[int][]string{
+	1: []string{
+		// cloud output v1: to be removed in the future
+		"K6_CLOUD_AGGREGATION_MIN_SAMPLES",
+		"K6_CLOUD_AGGREGATION_PERIOD",
+		"K6_CLOUD_AGGREGATION_WAIT_PERIOD",
+		"K6_CLOUD_METRIC_PUSH_INTERVAL",
+		"K6_CLOUD_MAX_METRIC_SAMPLES_PER_PACKAGE",
+		"K6_CLOUD_MAX_METRIC_PUSH_CONCURRENCY",
```

Comment:
Still open but we may address in a dedicated PR, maybe can we open a specific issue for it?

Reply:
It seems to work as is now: config override is being passed from the Cloud for v2 without error :shrug: So it doesn't look like an issue for k6-operator at least.

I suspect, I might end up removing support for decoding cloud output v1 before getting any updates about this topic :joy:

Reply:
`K6_CLOUD_MAX_METRIC_PUSH_CONCURRENCY` is not v2 but it is in the list for v1.

Reply:
Resolving as discussed in Slack.